packages/iTwinUI-react/stories/core/ComboBox.stories.tsx

```diff
@@ -9,12 +9,12 @@ import {
   ComboBox,
   ComboBoxProps,
   Label,
-  MenuItem,
   StatusMessage,
   SelectOption,
+  ProgressRadial,
 } from '../../src/core';
 import { CreeveyStoryParams } from 'creevey';
-import { SvgCamera } from '@itwin/itwinui-icons-react';
+import { SvgCamera, SvgSearch } from '@itwin/itwinui-icons-react';
 
```

Comment:
Disabled story takes a long time to load, whole window freezes and fans start to go wild.

Reply:
Any ideas what is the cause?

Reply:
idk. must be some storybook bug. I don't think it's related to this PR

Reply:
But why it was working previously? 😀

Reply:
it lags for me even without this PR, so I don't think it was working previously